README.md

```diff
@@ -17,14 +17,17 @@ designed as libraries (for programmatic use) with minimal command line interface
 
 ## Installation
 
-After cloning the Git repository, on the command line do execute
+To get setup with the OSS Review Toolkit, simply:
+
+1. Ensure the Java JDK (not the JRE as you need the `javac` compiler) is installed and the JAVA_HOME environment variable set.
+2. Clone this repository.
+3. Change into the repo directory on your machine and run
 
 ```
```

Comment:
I'd rather inline this into the numbered list item now, like `./gradlew installDist`. Otherwise the indentation of the sentence below starting with "to setup the build environment" is off.